### ISSUE TITLE
ConnectionError during download

### ISSUE THREAD

**Issue by edsu** [jjjake/internetarchive#174]
I'm seeing a ConnectionError when attempting to download with python v3.6.0 and requests v2.12.4:

```python
#!/usr/bin/env python

import internetarchive as ia

ia.download('2600magazine', verbose=True)
```

It runs for a few items and then throws this: 

```
 error downloading file 2600magazine/2600_1-10_jp2.zip, exception raised: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/home/ed/.pyenv/versions/2600/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 594, in urlopen
    chunked=chunked)
  File "/home/ed/.pyenv/versions/2600/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 391, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/ed/.pyenv/versions/2600/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 387, in _make_request
    httplib_response = conn.getresponse()
  File "/home/ed/.pyenv/versions/3.6.0/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/ed/.pyenv/versions/3.6.0/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/ed/.pyenv/versions/3.6.0/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
```

I've used internetarchive in the past and it has worked flawlessly (thank you!), so maybe something is up on the server side?

**Comment by jjjake** [comment]
This looks like an issue on the server side indeed. I am running your code above (on Python 3.6.0 & ia v1.1.0), and everything is looking good (I've downloaded about 150/928 files so far, including `2600magazine/2600_1-10_jp2.zip`). Can you retry and confirm whether the issue was transient or not for you?

Thanks for reporting the issue @edsu!

**Comment by jjjake** [comment]
I believe the issue here was with archive.org hanging up the phone before sending the complete response to the client. This connection would then get returned to the pool in a funky state (client-side), and reusing it causes this issue.

I believe forcing the connection to close after each request, rather than reusing the connection, resolves the problem. I think https://github.com/jjjake/internetarchive/commit/8fa4cd6db9301b552dbe8d98875d1b9437fae0d7 takes care of this.

That change was first released in v1.7.7. I'm closing this in hopes that this issue is resolved. Please reopen it if you continue to have issues.
